vmdb/app/apis/miqservices_ops.rb

```diff
@@ -244,6 +243,7 @@ def agent_config(hostId, config)
 
   def agent_register(xmlFile)
     ret = false
+    require 'miq-xml'
```

Comment:
Is there a reason we don't want to load it application-wide?  If we're confident that this code path is frequently used, then we'd benefit more by requiring up front (though it will slow boot time).  Otherwise, I'd go with the autoload solution (mainly because then we don't have to figure out which methods need the constant)